src/commands/cursorDown.tsx

```diff
@@ -91,7 +91,6 @@ const cursorDownShortcut: Command = {
       const isNextPathMulticursor = nextPath && isMulticursorPath(state, nextPath)
 
       dispatch([
-        setCursor({ path: nextPath, preserveMulticursor: true }),
```

Comment:
It doesn't seem correct to change `cursorDown` when the `cursorDown` behavior is not changing. Just the expansion behavior is changing.